WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -1474,8 +1473,8 @@ extension AztecPostViewController : Aztec.FormatBarDelegate {
                                                    selectedRowIndex index: Int?,
                                                    onSelect: OptionsTableViewController.OnSelectHandler?) {
         // Hide the input view if we're already showing these options
-        if let optionsViewController = optionsViewController, optionsViewController.options == options {
-            if presentedViewController != nil {
+        if let optionsViewController = optionsViewController ?? (presentedViewController as? OptionsTableViewController), optionsViewController.options == options {
```

Comment:
I think providing a default value if `optionsViewController` is nil doesn't really do anything because the if statement following only allows dismissal if the `self.optionsViewController` isn't nil.